### ISSUE TITLE
Does not support dynamic custom properties

### ISSUE THREAD

**Issue by artgud** [haftahave/serverless-ses-template#12]
Following expression is not supported by `@haftahave/serverless-ses-template` in serverless.yml

```
custom: ${file(../${opt:stage, self:provider.stage}.yml)}
```

Config file `sesTemplatesConfigFile` is always default one (`defaultSesTemplatesConfigFilePath`), because expression is not solved (remains `${file(../${opt:stage, self:provider.stage}.yml)}`):

```
this.sesTemplatesConfigFile = this.serverless.service.custom.sesTemplatesConfigFile
            || defaultSesTemplatesConfigFilePath;
```

Example of an external file (dev.yml):

```
...
sesTemplatesConfigFile: './email/config.js'
...
```

**Comment by Enase** [comment]
Thank for reporting! I will fix it soon. Issue relates to https://github.com/serverless/serverless/pull/3911

**Comment by Enase** [comment]
Fixed, feel free to use new version **1.2.5**.
